qiskit/providers/ibmq/experiment/constants.py

```diff
@@ -46,13 +46,9 @@ def __lt__(self, other: Any) -> bool:
             return self.ranking < other.ranking  # type: ignore[attr-defined]
         return NotImplemented
 
-    HUMAN_BAD = 'Human Bad', 1
-    COMPUTER_BAD = 'Computer Bad', 2
-    BAD = 'Bad', 3
-    NO_INFORMATION = 'No Information', 4
-    GOOD = 'Good', 5
-    COMPUTER_GOOD = 'Computer Good', 6
-    HUMAN_GOOD = 'Human Good', 7
+    BAD = 'Bad', 1
+    NO_INFORMATION = 'No Information', 2
```

Comment:
We are changing `NO_INFORMATION` to `UNKNOWN` in Qiskit to make it a single word. This class is being moved terra so it's not super important to make the change here, but if we're going to introduce breaking changes might as well do it once.

Reply:
Ack, will update today.